README.md

```diff
@@ -52,36 +52,37 @@ In addition, certain aspects can be overridden dynamically at runtime from a JSO
+| `STATUS_CHECK_INTERVAL_MS` | It defines how often (in ms) the tool updates internal status information (i.e. percentage of consumed messages) to expose outside on the corresponding HTTP endpoint.                                                                                                                                                                                                                                                                                                | `30000`                 |  |
+| `STATUS_TIME_WINDOW_MS` | It defines the sliding time window size (in ms) in which status information are sampled.                                                                                                                                                                                                                                                                                                                                                                              | `300000`                |  |
+| `DYNAMIC_CONFIG_FILE` | Location of an optional external config file that provides configuration at runtime.                                                                                                                                                                                                                                                                                                                                                                                  | empty                   |  |
+| `DYNAMIC_CONFIG_WATCHER_INTERVAL` | Interval that dynamic config file is examined for changes in content (in ms)                                                                                                                                                                                                                                                                                                                                                                                          | `30000`                 |  |
+| `ACTIVATE_TRACING` | Activate console tracing                                                                                                                                                                                                                                                                                                                                                                                                                                              | `false`                 |  |
```

Comment:
Need to add the Jaeger var here if we keep it.

Reply:
i put the env variables needed